compositor_chromium/src/v8/array.rs

```diff
@@ -0,0 +1,87 @@
+use crate::validated::{Validated, ValidatedError};
+
+use super::value::{V8Value, V8ValueError};
+
+pub struct V8Array(pub(super) Validated<chromium_sys::cef_v8value_t>);
+
+impl V8Array {
+    pub fn has(&self, index: usize) -> Result<bool, V8ArrayError> {
+        let inner = self.0.get()?;
```

Comment:
is this usefull for anything? len should be enough to verify if elements in array exists

Reply:
`len()` returns max size of an array. The array itself can have less elements than its `len`. I think the name is confusing. I will rename it